libs/langchain/langchain/output_parsers/openai_tools.py

```diff
@@ -22,8 +22,16 @@ class JsonOutputToolsParser(BaseGenerationOutputParser[Any]):
     """
     return_id: bool = False
     """Whether to return the tool call id."""
-    return_single: bool = False
-    """Whether to return only the first tool call."""
+    first_tool_only: bool = False
```

Comment:
breaking

Reply:
should probs alias

Reply:
Why is this breaking? is this used somewhere already?

Reply:
L76